src/AppLogin.tsx

```diff
@@ -180,13 +185,11 @@ const AppLogin = ({ toggleDarkMode }) => {
           <ItemVV2 alignSelf="stretch" alignItems="flex-start" margin={`0 0 ${GLOBALS.ADJUSTMENTS.MARGIN.VERTICAL} 0`}>
             {Object.keys(web3ConnectorsObj).map((name) => {
               const currentConnector = web3Connectors[name].obj;
-              const disabled = currentConnector === connector;
               const image = theme.scheme == 'light' ? web3Connectors[name].logolight : web3Connectors[name].logodark;
               const title = web3Connectors[name].title;
 
               return (
                 <LoginButton
```

Comment:
removing disable is not the best way to solve it since ig it is here for some reason